doc/source/api/general/data_transfer.rst

```diff
@@ -1,33 +1,38 @@
 .. _ref_data_transfer:
 
-Transfer data
-=============
+Transferring data between sessions
+==================================
 
-You use the `'transfer_case`` method to transfer a case or mesh file between
-instances of Fluent. You must specify a source instance and one or more
-destination instances.
+You use the ``transfer_case`` function to transfer a case or mesh file between
+PyFluent sessions. You must specify a source session and one or more
+destination sessions.
 
 Sample usage
 ------------
 
-To use the ``transfer_case`` method to transfer a mesh or a case file, you import the
-file and pass the source, destination instances, and the file type. You can either
-generate the files in the parent instance or just read the file.
-
-This example shows how you use the ``transfer_case`` method to read a mesh file in a
-pure meshing session and transfer it to a solver session.
+This example shows how you use the ``transfer_case`` function to read a mesh file in a
+meshing session and transfer it to a solver session.
```

Comment:
Add more rationale here about using pure-meshing session?? Probably hyperlink to the session details,  where all these sessions are explained in details.

Reply:
I'm thinking that for anything substantial like, we can add it to the _EPIC_. I will make sure that links to issues are added here before resolving each comment.

TODO create issue